bigtable/internal/testproxy/proxy.go

```diff
@@ -617,10 +626,7 @@ func (s *goTestProxyServer) ReadRow(ctx context.Context, req *pb.ReadRowRequest)
 		Row: &btpb.Row{},
 	}
 
-	if btc.perOperationTimeout != nil {
-		ct, _ := context.WithTimeout(ctx, btc.perOperationTimeout.AsDuration())
-		ctx = ct
-	}
+	ctx, _ = btc.timeout(ctx)
```

Comment:
Let's add a `defer cancel()` on all of these.

See https://stackoverflow.com/a/44394873/927514.